qabelbox/src/main/java/de/qabel/qabelbox/activities/MainActivity.java

```diff
@@ -228,6 +230,16 @@ private boolean uploadUri(Uri uri, String targetFolder) {
     protected void onCreate(Bundle savedInstanceState) {
 
         super.onCreate(savedInstanceState);
+        LocalQabelService service = QabelBoxApplication.getInstance().getService();
```

Comment:
yes, imho its save. the onbind method create a connection to a service. is the service not running, the service is started. if another component bind to service, he get only a instance to existing service. the service is destroyed after all components unbind from this. the service instance is all the same for all activities because the first code that execute is the qabelapplication class 
(they firstly initial the service). all other componets (splash, main) get only a instance. with the unbind in mainactivity we get no more the exception after with close the mainactivity.  the service is still running until the qableapplication class hold a instance to service (they calling not unbind)